### ISSUE TITLE
Add the ability to sort boolean values

### ISSUE THREAD

**Issue by mvila** [sindresorhus/sort-on#18]
Currently, invoking:

```
sortOn([{x: true}, {x: false}], 'x')
```

Is returning:

```
[{x: true}, {x: false}]
```

It would be nice to return the following instead:

```
[{x: false}, {x: true}]
```

Cheers! 😄

**Comment by sindresorhus** [comment]
Yeah, would be useful to support booleans, but I think `true` should be sorted first.

**Comment by mvila** [comment]
Hmm... Sorting `true` first seems counterintuitive to me, and it doesn't match the expected behavior that is the way sorting is handled in databases, spreadsheets, etc. `false` is usually considered as `0`, and `true` as `1`. Therefore, `false` should be sorted first.

**Comment by stevemao** [comment]
Because `true > false === true` so it should probably be `false` first.

**Comment by HitendraSinghRathore** [comment]
https://github.com/sindresorhus/sort-on/pull/26
Have added the fix in the PR.
The issue will be fixed once PR is merged.
